content/pl/user/signup.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Zarejestruj konto
```

Comment:
```suggestion
title: Zakładanie konta
```

Reply:
I feel the way I translated headlines/titles is just better for the ‘user guide' part of documentation, it feels more engaging to a user, although might be less accurate.